portfolio/src/main/webapp/js/chart.js

```diff
@@ -23,12 +24,30 @@ async function drawChart(request = (new Request('/data', {method: 'POST'}))) {
 
     const chart = new google.visualization.ColumnChart(document.getElementById('chart-container'));
     chart.draw(data, options);
+
+    if (pageNum <= 0) {
```

Comment:
The code is equivalent to 
```javascript
var previousButtonDisabled = pageNum <= 0;
document.getElementById('previous-button-top').disabled = disabled;
document.getElementById('previous-button-bottom').disabled = disabled;

var nextButtonDisabled = serverData.totalNumOfComments <= (pageNum+1)*document.getElementById('num-results').value;
document.getElementById('next-button-top').disabled = nextButtonDisabled;
document.getElementById('next-button-bottom').disabled = nextButtonDisabled;
```

Reply:
I see, this is definitely cleaner and more clear, thank you for the suggestion.